src/libraries/System.Net.HttpListener/src/System/Net/Windows/HttpListener.Windows.cs

```diff
@@ -508,7 +508,14 @@ public HttpListenerContext GetContext()
                 uint size = 4096;
                 ulong requestId = 0;
                 memoryBlob = new SyncRequestContext((int)size);
-                HttpListenerSession session = _currentSession!;
+                HttpListenerSession session = _currentSession;
```

Comment:
```suggestion
                HttpListenerSession? session = _currentSession;
```

Nit: I expect the build will show a nullability warning here since the field is nullable